packages/block-library/src/block/edit-panel/editor.scss

```diff
@@ -73,4 +73,14 @@
 		border-color: $light-opacity-light-800;
 		border-left-color: transparent;
 	}
+
+	.is-navigate-mode & {
+		border-color: $blue-medium-focus;
+		border-left-color: transparent;
+	}
+}
+
+.editor-block-list__layout .is-selected.is-navigate-mode .reusable-block-edit-panel {
```

Comment:
Could this one do with less specificity?

Reply:
Thanks for pointing this out. I just noticed that I left a CSS rule on lines 77-81 that doesn't actually do anything.

And yes, it looks like it can do with less specificity. I'll remove the `.editor-block-list__layout ` at the start of the rule.

In fact, it seems like a lot of the reusable block styles could have their specificity reduced, but I think I'll leave that for a future PR since it has a greater chance of causing regressions.